### DISCUSSION TITLE
Can't submit my edycational application

### DISCUSSION THREAD

**Discussion by egorgrin** [community/community#103096] [GitHub Education]
### Hi everyone,

I can't submit my edycational application for the reason of "Please select a different email address. We require applicants of Moscow University for Industry and Finance Synergy to use one of these school-issued email addresses to apply: - synergy.ru - synergyedu.ru"
The problem is - thy DO NON isuue such emails. And the heck am i supposed to do? This's funny

**Comment by gil-son** [comment]
You must to contact [Github Support](https://support.github.com/) 

I hope I was helpfull. Good luck!
